msg/FailsafeFlags.msg

```diff
@@ -49,6 +49,7 @@ bool vtol_fixed_wing_system_failure   # vehicle in fixed-wing system failure fai
 bool wind_limit_exceeded              # Wind limit exceeded
 bool flight_time_limit_exceeded       # Maximum flight time exceeded
 bool local_position_accuracy_low      # Local position estimate has dropped below threshold, but is currently still declared valid
+bool nav_failure_imminent_warned      # True if a warning for low
```

Comment:
This should not be in here if not used by the failsafe state machine.

Reply:
Okay moved it to be a global variable 9ee0b78e7964ef04494bf295d5d82d4db2e2f3ba